### ISSUE TITLE
Add way to get both nodes and relationships via helper methods

### ISSUE THREAD

**Issue by johnwunder** [neo4jrb/neo4j-core#172]
It's possible with a direct query, but it seems like you could have a method like .rels and .nodes on CypherNode that would return both the relationships and the nodes rather than one or the other.

**Comment by cheerfulstoic** [comment]
Good point.  I've normally been working with ActiveNode, but I've been using straight up `Query` lately and that would be handy
